### ISSUE TITLE
detective-postcss need update to 5.1.3

### ISSUE THREAD

**Issue by damingerdai** [angular/angular#48532]
### Which @angular/* package(s) are the source of the bug?

Don't known / other

### Is this a regression?

Yes

### Description

when i try to run `yarn` in my local to build the source code, i find the issue
```
error detective-postcss@5.1.1: The engine "node" is incompatible with this module. Expected version "12.x || 14.x || 16.x". Got "18.12.0"
error Found incompatible module.
```

now i use ` yarn config set ignore-engines true` to fix the issue, but i think we need upgrade `detective-postcss` lastest version to support node18

**Comment by JoostK** [comment]
Which version of Angular are you using? Angular 15 is required for Node 18 support.

**Comment by alan-agius4** [comment]
@JoostK, this is a matter of building the repo using Node 18.

**Comment by damingerdai** [comment]
> Which version of Angular are you using? Angular 15 is required for Node 18 support.

i just try to build the repo using node 18
```zsh
➜  angular git:(main) ✗ node -v
v18.12.0
➜  angular git:(main) ✗ yarn -v
1.22.17
➜  angular git:(main) ✗ uname -a
Linux DESKTOP-BH3E3T9 5.15.79.1-microsoft-standard-WSL2 #1 SMP Wed Nov 23 01:01:46 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by damingerdai** [comment]
i think angular should be built on node18

```diff
nodejs_register_toolchains(
    name = "nodejs",
+    node_version = "18.12.0",
-    node_version = "16.13.0",
)
```

**Comment by damingerdai** [comment]
it doesn't work

```
No nodejs is available for linux_amd64 at version 18.12.0
```

**Comment by JoostK** [comment]
Closing as the repo has been switched to use Node 18 so this should no longer be relevant.
